.php_cs

```diff
@@ -0,0 +1,9 @@
+<?php
+
+require_once './vendor/autoload.php';
```

Comment:
you should use ``__DIR__`` instead of relying on the current working directory being the root of the project